docs/tutorials/Containers.md

```diff
@@ -0,0 +1,475 @@
+## Containers
+
+Containers are self-contained software archives, that hold everything from the tool you want to run, to libraries and runtimes it requires, and the operating system it runs on.
+ 
```

Comment:
It doesn't quite hit the nail to call it a self-contained software archive - it's really more of an isolated set of system processes, and another layer of abstraction. But for the purposes of understanding, I see what you are getting at. What about:

> Containers are encapsulated environments that include an operating system, libraries, and software. If you have a host machine running Centos, you can run an isolated container with Ubuntu 18.04. At a high level, it's useful to think of a container as a program or binary.

Reply:
I changed it to your suggestion :)